rules/cloud/azure/azure_ad_failed_auth_from_countries_you_do_not_operate_out_of.yml

```diff
@@ -0,0 +1,22 @@
+title: Failed Authentications From Countries You Do Not Operate Out Of
+id: 28870ae4-6a13-4616-bd1a-235a7fad7458
+status: experimental
+author: MikeDuddington, '@dudders1'
+date: 2022/07/28
+description: Detect failed authentications from countries you do not operate out of.
+references:
+  - https://docs.microsoft.com/en-gb/azure/active-directory/fundamentals/security-operations-user-accounts
+logsource:
+  product: azure
+  service: signinlogs
+detection:
+  selection:
+    Status|not: 'Success'
```

Comment:
HI `not` is not in https://github.com/SigmaHQ/sigma/wiki/Specification#transformations
Here you can use a global not
```yaml
  selection:
    Status: 'Success'
    Location|contains: 'Countries you DO operate out of e,g GB, use OR for mulitple>'
  condition: not selection 
```